src/modules/global_css/index.js

```diff
@@ -96,6 +99,16 @@ class GlobalCSSModule {
     toggleLeftSideChat() {
         $('body').toggleClass('swap-chat', settings.get('leftSideChat'));
     }
+
+    attachTheatreListener() {
+        const playerService = App.__container__.lookup('service:persistentPlayer');
+        if (!playerService || !playerService.playerComponent || !playerService.playerComponent.player) return;
+        playerService.playerComponent.player.addEventListener('theatrechange', state => {
+            if (settings.get('darkenedMode') === true) return;
```

Comment:
you are not handling the case where the user exits theater mode.. we need to disable darkened mode